Batteries/Data/UInt.lean

```diff
@@ -42,7 +42,7 @@ theorem UInt8.max_def (x y : UInt8) : max x y = if x ≤ y then y else x := rfl
 
 theorem UInt8.min_def (x y : UInt8) : min x y = if x ≤ y then x else y := rfl
 
-theorem UInt8.toNat_max (x y : UInt8) : (max x y).toNat = max x.toNat y.toNat := by
+theorem UInt8.max_toNat (x y : UInt8) : (max x y).toNat = max x.toNat y.toNat := by
```

Comment:
I think the old name was correct here, this is `toNat (max x y)`?

Reply:
Yes, I agree. But UInt uses the reverse convention everywhere so there appears to be a local convention.

Reply:
There's a few of these (e.g. `Array.data`). They're probably all legacy names but they're pretty solidly rooted.

Reply:
I un-reverted. Looks like it's only `zero_toNat`, `div_toNat`, `mod_toNat` that need fixing in core, so the roots are not that deep.

Reply:
Fixing naming convention in core: [lean4#4514](https://github.com/leanprover/lean4/pull/4514).